### ISSUE TITLE
Question about the results obtained in the training.

### ISSUE THREAD

**Issue by RSanchez996** [georkara/Chargym-Charging-Station#5]
Hi, I have been testing and retraining the neural network for a few days now, and I can't get the results to improve. The best result I have obtained with the mean reward with DDPG is -24. I have tried adjusting the hyperparameters as detailed in the paper, but the behavior is still not improving. Could you give me a little guideline to try to improve the training results and get my results closer to yours? They really seem to look very good!
Great job!

**Comment by 160582221** [comment]
Hi，I have the same problem. Can you tell me how you solved it?

**Comment by RSanchez996** [comment]
Hi, unfortunately, with the Stable_Baselines3 training algorithms I have not been able to improve the rewards. I still have the same problem.

**Comment by 160582221** [comment]
Hi, thank you for your response. I'm experiencing slow training speed while training the DDPG algorithm. Have you encountered a similar problem?

**Comment by RSanchez996** [comment]
Hello,
Yes, correct, it is an algorithm that takes more computational load. If you have a GPU with CUDA, you can configure the training algorithm to use the GPU instead of the CPU and speed up the training slightly.

**Comment by RSanchez996** [comment]
Hello,
Yes, correct, it is an algorithm that takes more computational load. If you have a GPU with CUDA, you can configure the training algorithm to use the GPU instead of the CPU and speed up the training slightly.
